protocol/scripts/deploys/mainnet/v2/ramos/9999-test-ramos.ts

```diff
@@ -3,7 +3,7 @@ import { ethers } from "hardhat";
 import {
   ensureExpectedEnvvars,
   mine,
-} from "../../helpers";
+} from "../../../helpers";
 import { ContractInstances, connectToContracts } from "../contract-addresses";
 import { BigNumber } from "ethers";
 
```

Comment:
This file shouldn't be in the mainnet deploy - we won't be testing like (with real $$)